### ISSUE TITLE
loki and promtail binaries do not report semver version anymore (all platforms)

### ISSUE THREAD

**Issue by aliculPix4D** [grafana/loki#8055]
**Describe the bug**
`loki -version` and `promtail -version` stopped reporting the semver version (somewhere around v2.5.X). Instead only commit sha version is reported: `version HEAD-e0af1cc`.

**To Reproduce**
```
$ curl -O -L "https://github.com/grafana/loki/releases/download/v2.4.2/promtail-linux-amd64.zip"
$ unzip "promtail-linux-amd64.zip"
$ ./promtail-linux-amd64 -version
promtail, version 2.4.2 (branch: HEAD, revision: 525040a32)
    ....
```
repeating the same with v2.7.1 results in:
```
$ ./promtail-linux-amd64 -version
promtail, version HEAD-e0af1cc (branch: HEAD, revision: e0af1cc8
```
reporting the commit sha instead of a semver version.

**Expected behavior**
Semver version is correctly reported, since a lot of our automation scripts (i.e ansible) depend on the semver version parsed from `promtail -version`

**Comment by ashleykleynhans** [comment]
This is fixed in version 2.7.3.

**Comment by aliculPix4D** [comment]
Thanks for fixing this so quickly.
